CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Views/EditProfile.xaml.cs

```diff
@@ -0,0 +1,35 @@
+﻿using CheckDrive.Mobile.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace CheckDrive.Mobile.Views
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class EditProfile : ContentPage
+    {
+        private readonly ProfileViewModel _viewModel;
+        public EditProfile()
+        {
+            InitializeComponent();
+
+            _viewModel = new ProfileViewModel();
+
+            BindingContext = _viewModel;
+        }
+        protected override async void OnAppearing()
+        {
+            if (BindingContext is ProfileViewModel vm)
+            {
+                await vm.LoadProfileDataAsync();
+            }
```

Comment:
we already have it in **_viewModel** no need for casting.

`await _viewModel.LoadProfileDataAsync();`